awx/main/tasks/jobs.py

```
@@ -532,6 +543,90 @@ def final_run_hook(self, instance, status, private_data_dir):
+                f"Successfully obtained workload identity token for job {instance.id}. "
+                f"JWT length: {len(response.jwt)} characters"
+            )
+            logger.debug(f"JWT token: {response.jwt[:50]}...")
+            
+            # TODO (AAP-62694): Store JWT for use by credential plugins
+            # For now, we just log that we got it
+            
+        except TokenRequestError as e:
+            logger.error(
+                f"Failed to obtain workload identity token for job {instance.id}: {e}"
+            )
```

Ruff (0.15.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)